workshops/dodge/README.md

```diff
@@ -704,6 +704,7 @@ Infinite possibilities await you! [p5.js documentation](http://p5js.org/referenc
 - [Patrick Dodge](http://katielattaa.github.io/dodge/)
 - [Dodge with Levels](https://gabriellazrak.github.io/dodge/)
 - [Collect Poo](https://jocyhe.github.io/collectpoo/index.html)
+- [Example of a game online](http://www.roundgames.com/game/Ragdoll+Laser+Dodge)
```

Comment:
The other examples are also "games online". What if we call this "Ragdoll Laser Dodge".